examples/framesjs-starter/app/debug/hub/[...hubPath]/route.ts

```diff
@@ -28,6 +30,33 @@ export async function GET(
   request: NextRequest,
   { params: { hubPath } }: { params: { hubPath: string[] } }
 ) {
+  // Check if the request needs to be mocked
+  // Open json config file
+  // Check if the request is in the json file
+  // If it is, return the response from the json file
+  // If it is not, forward the request to the real hub
+
+  try {
+    // Only available in local development
+    const file = path.join(process.cwd(), "app", "debug", "mocks.json");
```

Comment:
this is going to break in deployment - and this debugger is also deployed. any reason we can't just use local storage?

Reply:
the mock hub doesn't have access to localStorage and is called asynchronously by the app

other options we have are redis, kv store, or redis

a file is the easiest way to do this without over engineering and catering to most of our userbase who are using the debugger locally

Reply:
the hosted debugger can't force apps to use its mock hub either, so it doesn't make sense for this to be available there

Reply:
hm ok that makes sense, can we make sure this feature is disabled (maybe disable checkboxes & say use local debugger by cloning frames starter (url) to use this feature)

Reply:
can do

Reply:
<img width="551" alt="Screenshot 2024-02-16 at 14 25 14" src="https://github.com/framesjs/frames.js/assets/5469870/b55bbf66-8267-4abf-846a-d69daca4c271">


links to https://framesjs.org/#1-clone-the-framesjs-starter-template-with-local-debugger